docs/content/preview/deploy/multi-dc/async-replication/async-transactional-switchover.md

```diff
@@ -0,0 +1,181 @@
+        -certs_dir_name <dir_name> \
+        wait_for_replication_drain 56bf794172da49c6804cbab59b978c7e,..,..<comma_separated_list_of_stream_ids> 1665548814177072
+    ```
+
+    ```output
+    All replications are caught-up.
+    ```
+
+    Note that this step may take some time.
+
+    If you get output similar to the following, then it is not yet safe to switch Standby (B) to active:
+
+    ```output
+    Found undrained replications:
+    - Under Stream b7306a8068484634801fd925a41f17f6:
+      - Tablet: 04dd056e79a14f48aac3bdf196546fd0
+      - Tablet: ebc07bd926b647a5ba8cea41421ec7ca
+      - Tablet: 3687d4d51cda4f0394b4c36aa66f4ffd
+      - Tablet: 318745aee32c463c81d4242924d679a2
+    ```
+
+1. Wait until [xCluster safe time](../async-transactional-setup/#verify-replication) on Standby (B) is greater than the current time obtained on Primary (A).
+
+1. Use `change_xcluster_role` to promote Standby (B) to ACTIVE as follows.
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses <B_master_addresses> \ 
+        -certs_dir_name <dir_name> \
+        change_xcluster_role ACTIVE
+    ```
+
+1. Delete the replication from A to B as follows:
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses <B_master_addresses> \
+        delete_universe_replication <A_universe_uuid>_<replication_name>
+    ```
+
+### Set up replication
+
+In the second stage, set up replication from the new Primary (B) universe as follows:
+
+1. Run `bootstrap_cdc_producer` command on the new Primary (B) for all tables involved in replication and capture the bootstrap IDs. This is a preemptive step to be used during the failover workflow while setting up replication from B to A. This prevents the WAL from being garbage collected and lets A catch up.
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses <B_master_addresses> 
+        -certs_dir_name <cert_dir> \
+        bootstrap_cdc_producer <comma_separated_B_table_ids>
+    ```
+
+1. Resume the application traffic on the new Primary (B).
+
+1. Prepare certificates on all nodes of A for replication setup:
+
+    - Create a directory for the replication certificate:
+
+        ```sh
+        mkdir -p <home>/yugabyte-tls-producer/<B_universe_uuid>_<replicaten_name>
+        ```
+
+    - Copy the certificates:
+
+        ```sh
+        cp -rf <home>/yugabyte-tls-producer/<B_universe_uuid> <home>/yugabyte-tls-producer/<B_universe_uuid>_<replicaten_name>
+        ```
+
+    Repeat these steps on all A nodes.
+
+1. Set up replication from B to A using the bootstrap IDs obtained previously.
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses <A_master_addresses> \
+        -certs_dir_name <cert_dir> \
+        setup_universe_replication \
+        <B_universe_uuid>_<replication_name> \
+        <B_master_addresses> \
+        <comma_separated_B_table_ids>  \
+        <comma_separated_B_bootstrap_ids> transactional
+    ```
+
+1. Use `change_xcluster_role` to demote A to STANDBY:
```

Comment:
```suggestion
1. Use `change_xcluster_role` to demote A to Standby:
```

Reply:
I think capitalizing Standby in the code block is enough. Its confusing otherwise.